ngraph/python/requirements_test.txt

```diff
@@ -1,10 +1,13 @@
 flake8
+flake8-bugbear
 flake8-comprehensions
 flake8-docstrings
 flake8-quotes
+mypy
 onnx
 pydocstyle
 pytest
+pytest-xdist
 retrying
 tox
-wheel
+wheel==0.35.1
```

Comment:
Why do we need to freeze `wheel` version?

Reply:
in general all versions should be fixed. It helps to avoid incompatibility issues while building old releases, e.g. LTS